src/style.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+/* .block {
+  padding: 22px;
+} */
```

Comment:
We should remove commented code, to keep the codebase as clean as possible

Reply:
ok, I deleted comments at the moment.